examples/swagger/swagger/public/definitions/InterestedParty.yaml

```diff
@@ -0,0 +1,11 @@
+allOf:
+  - $ref: 'InvolvedParty.yaml'
```

Comment:
I think it best if we move to a different domain as Darrell suggested, or something meaningless, like `TypeA`, `TypeB`, `TypeR` ...